src/main/java/com/intuit/graphql/orchestrator/resolverdirective/FieldResolverDirectiveUtil.java

```diff
@@ -124,37 +125,35 @@ public static boolean isObjectOrInterfaceType(TypeDefinition typeDefinition) {
 
   public static List<FieldResolverContext> createFieldResolverContexts(
       TypeDefinition typeDefinition, XtextGraph xtextGraph) {
-
     List<FieldDefinition> childFields = getFieldDefinitions(typeDefinition);
-    return childFields.stream()
```

Comment:
I do not think this class needs to change for the issue

Reply:
FieldResolverTransformerPreMerge validates each directive here

Reply:
@vijayakinbox I don't think this is the right class to implement .  I would suggest create a Validator Class that 
1. accept `List<Directive>` as input.  See below line 131.
2. performs validation
3. throw exception if validation fails.

Reply:
The validator class should be enough for this issue, i.e. not integrated yet but with unit test.